e2e_test/batch/decribe_fragment.slt

```diff
@@ -0,0 +1,38 @@
+control substitution on
+
+# Test DESCRIBE FRAGMENT with different options
+
+# Setup test table and materialized view
+statement ok
+CREATE TABLE test_table (id INT PRIMARY KEY, value INT);
+
+statement ok
+CREATE MATERIALIZED VIEW test_table_mv AS
+SELECT id, SUM(value) as total FROM test_table GROUP BY id;
+
+system ok
+fragment_id=$(psql -tAc "SELECT fragment_id FROM fragment LIMIT 1")
+psql_validate.py --db $__DATABASE__ --sql "DESCRIBE FRAGMENT $fragment_id" \
+--expected 'Fragment % (Table %)
+Distribution Type: %
+Parallelism: %
+VNode Count: %
+State Tables: %
+Upstream Fragments: %
+(empty)
+Stream Plan:
+StreamMaterialize { columns: [id, value], stream_key: [id], pk_columns: [id], pk_conflict: Overwrite }
+├── output: [ id, value ]
+├── stream key: [ id ]
+└── StreamUnion { all: true } { output: [ id, value ], stream key: [] }
+    └── MergeExecutor { output: [ id, value ], stream key: [] }'
+
```

Comment:
You need to add a newline here, otherwise the subsequent section will be considered as part of the output as well.
```suggestion


```

Reply:
the test still hangs forever in my local setup and sadly the logs from the python code is not emitted as well :(